### PR TITLE
Add documentation for IPv6 in ipsubnet filter. Add unit tests

### PR DESCRIPTION
**Pull request by sebastien-rosset**
##### SUMMARY

1. Add example for IPv6 in `ipsubnet` filter documentation.
2. Fix typo in `ipsubnet` documentation.
3. Add unit tests for `ipsubnet` filter.

 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`ipsubnet` filter

##### ADDITIONAL INFORMATION

**Analysis**

1. In the netaddr module, there should be some documentation and protection to prevent a very large 2 ^ N loop. https://github.com/netaddr/netaddr/issues/241
2. In ansible.utils, the `ipsubnet` filter should not construct a list of subnets, then get subnet at index X. Instead, the subnet can be constructed using bit operations.

```paste below

```

### CODE REVIEW

File: tests/unit/plugins/filter/test_ipsubnet.py [hunk 1]
```diff
@@ -94,3 +94,73 @@ def test_ipvsubnet_filter_rank_address_in_subnet3(self):
...
+            [
+                ["", "2600:1f1c:1b3:8f00:::/56", "64", "0"],
+                False,
+            ],  # Too many colons
+            [
+                ["", "2600:1f1c:1b3:8f00::/56", "129", "0"],
+                False,
+            ],  # Must be /128 or less
+            [
+                ["", "2600:1f1c:1b3:8f00::/56", "120", "0"],
```
Comment (sebastien-rosset) [inline tests/unit/plugins/filter/test_ipsubnet.py:153]: This unit test with a `/120` prefix length causes the `ipsubnet` filter to use 100% CPU utilization.

**Comment by ashwini-mhatre** [comment]
@sebastien-rosset could you please open those bugs as separate issues?

File: tests/unit/plugins/filter/test_ipsubnet.py (hunk 1 above)
Comment (ashwini-mhatre) [reply tests/unit/plugins/filter/test_ipsubnet.py:153]: can you open another issue for this and in this prs only doc changes and other uts?

Comment (sebastien-rosset) [reply tests/unit/plugins/filter/test_ipsubnet.py:153]: You mean another issue in addition to #132 ?

File: changelogs/fragments/add_ipfilter_doc_ut.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
```
Comment (sebastien-rosset) [inline changelogs/fragments/add_ipfilter_doc_ut.yaml:2]:
```suggestion
doc_changes:
```

File: plugins/plugin_utils/base/ipaddr_utils.py [hunk 3]
```diff
@@ -368,7 +368,7 @@ def _wrap_query(v, vtype, value):
 
 
 def ipaddr(value, query="", version=False, alias="ipaddr"):
-    """ Check if string is an IP address or network and filter it """
+    """Check if string is an IP address or network and filter it"""
```
Comment (sebastien-rosset) [inline plugins/plugin_utils/base/ipaddr_utils.py:371]:
```suggestion
    """ Check if string is an IP address or network and filter it """
```

Comment (sebastien-rosset) [inline plugins/plugin_utils/base/ipaddr_utils.py]:
```suggestion
    """ Check if string is an IP address or network and filter it """
```

File: plugins/plugin_utils/base/ipaddr_utils.py [hunk 4]
```diff
@@ -689,7 +689,7 @@ def _win_query(v):
 
 # ---- HWaddr / MAC address filters ----
 def hwaddr(value, query="", alias="hwaddr"):
-    """ Check if string is a HW/MAC address and filter it """
+    """Check if string is a HW/MAC address and filter it"""
```
Comment (sebastien-rosset) [inline plugins/plugin_utils/base/ipaddr_utils.py:692]:
```suggestion
    """ Check if string is a HW/MAC address and filter it """
```

Comment (sebastien-rosset) [inline plugins/plugin_utils/base/ipaddr_utils.py]:
```suggestion
    """ Check if string is a HW/MAC address and filter it """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 5]
```diff
@@ -58,7 +58,7 @@ def setUp(self):
 
     @staticmethod
     def _load_fixture(filename):
-        """ Load a fixture from the filesystem
+        """Load a fixture from the filesystem
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:61]:
```suggestion
        """ Load a fixture from the filesystem
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 6]
```diff
@@ -72,23 +72,20 @@ def _load_fixture(filename):
             return fhand.read()
 
     def test_fn_debug(self):
-        """ Confirm debug doesn't fail and return None
-        """
+        """Confirm debug doesn't fail and return None"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:75]:
```suggestion
        """ Confirm debug doesn't fail and return None
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 7]
```diff
@@ -72,23 +72,20 @@ def _load_fixture(filename):
...
         msg = "some message"
         result = self._plugin._debug(msg)
         self.assertEqual(result, None)
 
     def test_fn_ail_json(self):
-        """ Confirm fail json replaces basic.py in msg
-        """
+        """Confirm fail json replaces basic.py in msg"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:81]:
```suggestion
        """ Confirm fail json replaces basic.py in msg
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 8]
```diff
@@ -72,23 +72,20 @@ def _load_fixture(filename):
...
         msg = "text (basic.py)"
         with self.assertRaises(Exception) as error:
             self._plugin._fail_json(msg)
         self.assertEqual("text cli_parse", str(error.exception))
 
     def test_fn_check_argspec_pass(self):
-        """ Confirm a valid argspec passes
-        """
+        """Confirm a valid argspec passes"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:88]:
```suggestion
        """ Confirm a valid argspec passes
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 9]
```diff
@@ -102,8 +99,7 @@ def test_fn_check_argspec_pass(self):
         self.assertEqual(valid, True)
 
     def test_fn_check_argspec_fail_no_test_or_command(self):
-        """ Confirm failed argpsec w/o text or command
-        """
+        """Confirm failed argpsec w/o text or command"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:102]:
```suggestion
        """ Confirm failed argpsec w/o text or command
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 10]
```diff
@@ -122,8 +118,7 @@ def test_fn_check_argspec_fail_no_test_or_command(self):
         )
 
     def test_fn_check_argspec_fail_no_parser_name(self):
-        """ Confirm failed argspec no parser name
-        """
+        """Confirm failed argspec no parser name"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:121]:
```suggestion
        """ Confirm failed argspec no parser name
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 11]
```diff
@@ -122,8 +119,7 @@ def test_fn_check_argspec_fail_no_test_or_command(self):
...
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py]:
```suggestion
        """ Confirm failed argspec no parser name
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 12]
```diff
@@ -137,8 +133,7 @@ def test_fn_check_argspec_fail_no_parser_name(self):
         )
 
     def test_fn_extended_check_argspec_parser_name_not_coll(self):
-        """ Confirm failed argpsec parser not collection format
-        """
+        """Confirm failed argpsec parser not collection format"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:137]:
```suggestion
        """ Confirm failed argpsec parser not collection format
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 13]
```diff
@@ -165,8 +161,7 @@ def test_fn_extended_check_argspec_missing_tpath_or_command(self):
         )
 
     def test_fn_load_parser_pass(self):
-        """ Confirm each each of the parsers loads from the filesystem
-        """
+        """Confirm each each of the parsers loads from the filesystem"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:165]:
```suggestion
        """ Confirm each each of the parsers loads from the filesystem
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 14]
```diff
@@ -151,7 +147,7 @@ def test_fn_extended_check_argspec_parser_name_not_coll(self):
         self.assertIn("including collection", self._plugin._result["msg"])
 
     def test_fn_extended_check_argspec_missing_tpath_or_command(self):
-        """ Confirm failed argpsec missing template_path
+        """Confirm failed argpsec missing template_path
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:151]:
```suggestion
        """ Confirm failed argpsec missing template_path
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 15]
```diff
@@ -179,8 +174,7 @@ def test_fn_load_parser_pass(self):
             self.assertTrue(callable(parser.parse))
 
     def test_fn_load_parser_fail(self):
-        """ Confirm missing parser fails gracefully
-        """
+        """Confirm missing parser fails gracefully"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:178]:
```suggestion
        """ Confirm missing parser fails gracefully
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 16]
```diff
@@ -191,7 +185,7 @@ def test_fn_load_parser_fail(self):
         self.assertIn("No module named", self._plugin._result["msg"])
 
     def test_fn_set_parser_command_missing(self):
-        """ Confirm parser/command is set if missing
+        """Confirm parser/command is set if missing
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:189]:
```suggestion
        """ Confirm parser/command is set if missing
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 17]
```diff
@@ -216,32 +213,28 @@ def test_fn_set_parser_command_present(self):
         )
 
     def test_fn_set_parser_command_absent(self):
-        """ Confirm parser/command is not added
-        """
+        """Confirm parser/command is not added"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:217]:
```suggestion
        """ Confirm parser/command is not added
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 18]
```diff
@@ -204,8 +202,7 @@ def test_fn_set_parser_command_missing(self):
         )
 
     def test_fn_set_parser_command_present(self):
-        """ Confirm parser/command is not changed if provided
-        """
+        """Confirm parser/command is not changed if provided"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:206]:
```suggestion
        """ Confirm parser/command is not changed if provided
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 19]
```diff
@@ -216,32 +214,28 @@ def test_fn_set_parser_command_present(self):
...
-        """
+        """Confirm parser/command is not added"""
         self._plugin._task.args = {"parser": {}}
         self._plugin._set_parser_command()
         self.assertNotIn("command", self._plugin._task.args["parser"])
 
     def test_fn_set_text_present(self):
-        """ Check task args text is set to stdout
-        """
+        """Check task args text is set to stdout"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:224]:
```suggestion
        """ Check task args text is set to stdout
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 20]
```diff
@@ -577,8 +557,7 @@ class CliParser(CliParserBase):
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_net_device_error(self, mock_rpc):
-        """ Check full module run mock error from network device
-        """
+        """Check full module run mock error from network device"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:561]:
```suggestion
        """ Check full module run mock error from network device
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 21]
```diff
@@ -542,7 +522,7 @@ def parse(self, *_args, **kwargs):
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_pass_missing_parser_in_parser(self, mock_rpc):
-        """ Check full module run using parser w/o
+        """Check full module run using parser w/o
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:526]:
```suggestion
        """ Check full module run using parser w/o
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 22]
```diff
@@ -505,7 +485,7 @@ def test_fn_run_fail_missing_parser(self):
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_pass_missing_parser_constants(self, mock_rpc):
-        """ Check full module run using parser w/o
+        """Check full module run using parser w/o
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:489]:
```suggestion
        """ Check full module run using parser w/o
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 23]
```diff
@@ -495,8 +476,7 @@ def test_fn_run_fail_command(self):
         self.assertEqual(result, expected)
 
     def test_fn_run_fail_missing_parser(self):
-        """Confirm clean fail with missing parser
-        """
+        """Confirm clean fail with missing parser"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:480]:
```suggestion
        """ Confirm clean fail with missing parser
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 24]
```diff
@@ -470,8 +452,7 @@ def test_fn_run_fail_argspec(self):
         self.assertIn("including collection", self._plugin._result["msg"])
 
     def test_fn_run_fail_command(self):
-        """ Confirm clean fail with rc 1
-        """
+        """Confirm clean fail with rc 1"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:456]:
```suggestion
        """ Confirm clean fail with rc 1
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 25]
```diff
@@ -456,8 +439,7 @@ def test_fn_run_pass_wo_fact(self, mock_rpc):
         self.assertNotIn("ansible_facts", result)
 
     def test_fn_run_fail_argspec(self):
-        """ Check full module run with invalid params
-        """
+        """Check full module run with invalid params"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:443]:
```suggestion
        """ Check full module run with invalid params
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 26]
```diff
@@ -431,8 +415,7 @@ def test_fn_run_pass_w_fact(self, mock_rpc):
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_pass_wo_fact(self, mock_rpc):
-        """ Check full module run with valid params
-        """
+        """Check full module run with valid params"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:419]:
```suggestion
        """ Check full module run with valid params
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 27]
```diff
@@ -394,16 +380,14 @@ def test_fn_run_command_network(self, mock_rpc):
...
+        """Check run command for network"""
         self._plugin._task.args = {"command": None}
         result = self._plugin._run_command()
         self.assertIsNone(result)
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_pass_w_fact(self, mock_rpc):
-        """ Check full module run with valid params
-        """
+        """Check full module run with valid params"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:391]:
```suggestion
        """ Check full module run with valid params
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 28]
```diff
@@ -394,16 +380,14 @@ def test_fn_run_command_network(self, mock_rpc):
         self.assertEqual(self._plugin._result["stdout_lines"], [expected])
 
     def test_fn_run_command_not_specified(self):
-        """ Check run command for network
-        """
+        """Check run command for network"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:384]:
```suggestion
        """ Check run command for network
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 29]
```diff
@@ -381,8 +368,7 @@ def test_fn_run_command_lx_rc1(self):
 
     @patch("ansible.module_utils.connection.Connection.__rpc__")
     def test_fn_run_command_network(self, mock_rpc):
-        """ Check run command for network
-        """
+        """Check run command for network"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:372]:
```suggestion
        """ Check run command for network
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 30]
```diff
@@ -363,8 +351,7 @@ def test_fn_run_command_lx_rc0(self):
         self.assertEqual(self._plugin._result["stdout_lines"], response)
 
     def test_fn_run_command_lx_rc1(self):
-        """ Check run command for non network
-        """
+        """Check run command for non network"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:355]:
```suggestion
        """ Check run command for non network
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 31]
```diff
@@ -324,31 +316,27 @@ def test_fn_get_template_contents_missing(self):
...
-        """
+        """Test the removal of stdout and stdout_lines from the _result"""
         self._plugin._prune_result()
         self.assertNotIn("stdout", self._plugin._result)
         self.assertNotIn("stdout_lines", self._plugin._result)
 
     def test_fn_run_command_lx_rc0(self):
-        """ Check run command for non network
-        """
+        """Check run command for non network"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:340]:
```suggestion
        """ Check run command for non network
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 32]
```diff
@@ -324,31 +316,27 @@ def test_fn_get_template_contents_missing(self):
...
         self._plugin._result["stdout"] = "abc"
         self._plugin._result["stdout_lines"] = "abc"
         self._plugin._prune_result()
         self.assertNotIn("stdout", self._plugin._result)
         self.assertNotIn("stdout_lines", self._plugin._result)
 
     def test_fn_prune_result_not_exist(self):
-        """ Test the removal of stdout and stdout_lines from the _result
-        """
+        """Test the removal of stdout and stdout_lines from the _result"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:334]:
```suggestion
        """ Test the removal of stdout and stdout_lines from the _result
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 33]
```diff
@@ -324,31 +316,27 @@ def test_fn_get_template_contents_missing(self):
...
-        """
+        """Check the none when template_path not specified"""
         self._plugin._task.args = {"parser": {}}
         result = self._plugin._get_template_contents()
         self.assertIsNone(result)
 
     def test_fn_prune_result_pass(self):
-        """ Test the removal of stdout and stdout_lines from the _result
-        """
+        """Test the removal of stdout and stdout_lines from the _result"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:326]:
```suggestion
        """ Test the removal of stdout and stdout_lines from the _result
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 34]
```diff
@@ -324,8 +317,7 @@ def test_fn_get_template_contents_missing(self):
         )
 
     def test_fn_get_template_contents_not_specified(self):
-        """ Check the none when template_path not specified
-        """
+        """Check the none when template_path not specified"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:320]:
```suggestion
        """ Check the none when template_path not specified
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 35]
```diff
@@ -314,8 +308,7 @@ def test_fn_get_template_contents_pass(self):
         self.assertEqual(result, contents)
 
     def test_fn_get_template_contents_missing(self):
-        """ Check the retrieval of the template contents
-        """
+        """Check the retrieval of the template contents"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:311]:
```suggestion
        """ Check the retrieval of the template contents
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 36]
```diff
@@ -302,8 +297,7 @@ def test_fn_update_template_path_mock_find_needle(self):
         )
 
     def test_fn_get_template_contents_pass(self):
-        """ Check the retrieval of the template contents
-        """
+        """Check the retrieval of the template contents"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:300]:
```suggestion
        """ Check the retrieval of the template contents
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 37]
```diff
@@ -284,7 +279,7 @@ def test_fn_update_template_path_not_exist_os(self):
         )
 
     def test_fn_update_template_path_mock_find_needle(self):
-        """ Check the creation of the template_path
+        """Check the creation of the template_path
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:282]:
```suggestion
        """ Check the creation of the template_path
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 38]
```diff
@@ -269,7 +264,7 @@ def test_fn_update_template_path_not_exist(self):
         )
 
     def test_fn_update_template_path_not_exist_os(self):
-        """ Check the creation of the template_path if
+        """Check the creation of the template_path if
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:267]:
```suggestion
        """ Check the creation of the template_path if
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 39]
```diff
@@ -254,7 +249,7 @@ def test_fn_os_from_task_vars(self):
             self.assertEqual(result, check[2])
 
     def test_fn_update_template_path_not_exist(self):
-        """ Check the creation of the template_path if
+        """Check the creation of the template_path if
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:252]:
```suggestion
        """ Check the creation of the template_path if
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 40]
```diff
@@ -216,32 +215,28 @@ def test_fn_set_parser_command_present(self):
...
+        """Check task args text is set to stdout"""
         self._plugin._result["stdout"] = None
         self._plugin._task.args = {}
         self._plugin._set_text()
         self.assertNotIn("text", self._plugin._task.args)
 
     def test_fn_os_from_task_vars(self):
-        """ Confirm os is set based on task vars
-        """
+        """Confirm os is set based on task vars"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:239]:
```suggestion
        """ Confirm os is set based on task vars
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 41]
```diff
@@ -216,32 +215,28 @@ def test_fn_set_parser_command_present(self):
...
         expected = "output"
         self._plugin._result["stdout"] = expected
         self._plugin._task.args = {}
         self._plugin._set_text()
         self.assertEqual(self._plugin._task.args["text"], expected)
 
     def test_fn_set_text_absent(self):
-        """ Check task args text is set to stdout
-        """
+        """Check task args text is set to stdout"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py:232]:
```suggestion
        """ Check task args text is set to stdout
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 42]
```diff
@@ -324,8 +323,7 @@ def test_fn_get_template_contents_missing(self):
         )
 
     def test_fn_get_template_contents_not_specified(self):
-        """ Check the none when template_path not specified
-        """
+        """Check the none when template_path not specified"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py]:
```suggestion
        """ Check the none when template_path not specified
        """
```

File: tests/unit/plugins/action/test_cli_parse.py [hunk 43]
```diff
@@ -72,8 +72,7 @@ def _load_fixture(filename):
             return fhand.read()
 
     def test_fn_debug(self):
-        """ Confirm debug doesn't fail and return None
-        """
+        """Confirm debug doesn't fail and return None"""
```
Comment (sebastien-rosset) [inline tests/unit/plugins/action/test_cli_parse.py]:
```suggestion
        """ Confirm debug doesn't fail and return None
        """
```

**Comment by ashwini-mhatre** [comment]
recheck

**Comment by ashwini-mhatre** [comment]
@sebastien-rosset This will be trivial change instead of minor_changes. could you please update changelog fragment?

**Comment by sebastien-rosset** [comment]
> @sebastien-rosset This will be trivial change instead of minor_changes. could you please update changelog fragment?

I don't see a "trivial" section in the changelogs/config.yaml file. Can you suggest a change?

https://github.com/ansible-collections/ansible.utils/blob/main/changelogs/config.yaml

**Comment by Qalthos** [comment]
> > @sebastien-rosset This will be trivial change instead of minor_changes. could you please update changelog fragment?
> 
> I don't see a "trivial" section in the changelogs/config.yaml file. Can you suggest a change?
> 
> https://github.com/ansible-collections/ansible.utils/blob/main/changelogs/config.yaml

That woufd be what `trivial_section_name: trivial` is for

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by Qalthos** [comment]
Replaced by #212

**Pull request closed without merging**